guestbook/frontend-service.yaml

```diff
@@ -6,6 +6,8 @@ metadata:
     app: guestbook
     tier: frontend
 spec:
+  # comment or delete the following line if you want to use a LoadBalancer
+  type: NodePort 
```

Comment:
we'll need to update instructions on https://cloud.google.com/container-engine/docs/tutorials/guestbook#expose_frontend_on_an_external_ip_address after merging this.

maybe at some point we should have 2 service files (or a single file that describes both a NodePort service as well as a LoadBalancer). This way, minikube people can get it to work as well as people trying it out on the cloud.